src/components/NcEmptyContent/NcEmptyContent.vue

```diff
@@ -187,7 +187,7 @@ export default {
 		 * Check if a description is given as either property or slot
 		 */
 		hasDescription() {
-			return this.description !== '' || this.$slots.description?.[0]
+			return this.description !== '' || this.$slots.description?.()?.[0]
```

Comment:
why would it be a function?

Reply:
In vue3 `$slots` is a record of `slotname -> function`, so for scoped slots you would pass in the arguments of the slot.
See https://vuejs.org/api/component-instance.html#slots

Reply:
If we need to check the content here, not if the slot just was passed, should we use `isSlotPopulated` here? Or just check `!!this.$slots.description` otherwise to not render description in computed?

Reply:
> Or just check `!!this.$slots.description` otherwise to not render description in computed?

That might be the best option here.

Reply:
Adjusted.